components/NavigationDrawer/src/MDCBottomDrawerPresentationController.h

```diff
@@ -87,4 +87,12 @@
  */
 @property(nonatomic, strong, nullable) UIColor *topHandleColor;
 
+/**
+ Expands the navigation drawer to maximum height for the content. If content is larger than the
+ screen bounds then animates to full screen.
```

Comment:
Doesn't it always just go to the full height of the window? is there a case where it reaches the maximum height of the content and it doesn't expand to full screen?

Reply:
Yes, if for example the content and header are a total height of 650 and the window is 800 then it will expand to ~81% of the screen.